datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -100,14 +100,6 @@ export class ChartEntity implements Entity<Chart> {
                     properties: {
                         defaultDirection: LineageDirection.Upstream,
                     },
-                    display: {
```

Comment:
why remove this?

Reply:
basicaly lineage tab will always be visible now? got it. that makes sensee. forget the above!